### DISCUSSION TITLE
php.log vs prod.log

### DISCUSSION THREAD

**Discussion by hethehe** [pimcore/pimcore#17201] [Q&A / Support] [answered]
What's is the idea behind prod.log and php.log?

https://github.com/pimcore/pimcore/blob/11.x/bundles/CoreBundle/config/pimcore/prod.yaml both are configured the same way. If you do not overwrite them they log exactly the same data in the prod environment.

Or do I misunderstand something?

**Comment by blankse** [comment, marked as answer]
@hethehe I also think that's nonsense. I created a PR: https://github.com/pimcore/pimcore/pull/17209

**Reply by blankse** [reply]
@hethehe The PR is merged. Will be fixed in the next minor version (11.4.0) of Pimcore.
